src/FeedIo/Feed.php

```diff
@@ -71,7 +71,7 @@ public function current()
      */
     public function next()
     {
-        return $this->items->next();
+        $this->items->next();
```

Comment:
From docblock:
```php
    /**
     * (PHP 5 &gt;= 5.0.0)<br/>
     * Move forward to next element
     * @link http://php.net/manual/en/iterator.next.php
     * @return void Any returned value is ignored.
     */
    public function next()
```
Returning anything when interface (and iterator documentation) says otherwise not only don't follow interface (with php < 7.0 it's only an agreement, when you get return types it'd be an `ErrorException`).

This should return void.